LightBlue/Hosted/HostedAzureLocalResourceSource.cs

```diff
@@ -1,12 +1,22 @@
-﻿using Microsoft.WindowsAzure.ServiceRuntime;
+﻿using LightBlue.Standalone;
+using Microsoft.WindowsAzure.ServiceRuntime;
 
 namespace LightBlue.Hosted
 {
     public class HostedAzureLocalResourceSource : IAzureLocalResourceSource
     {
         public IAzureLocalResource this[string index]
         {
-            get { return new HostedAzureLocalResource(RoleEnvironment.GetLocalResource(index)); }
+            get
+            {
+                if (RoleEnvironment.IsAvailable)
```

Comment:
I think it might be better if this throws if the role environment isn't available? ie if something requires local disk access, and we don't have a facility to provide it...